staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -240,6 +240,29 @@ type VirtualMachineInstanceStatus struct {
 	TopologyHints *TopologyHints `json:"topologyHints,omitempty"`
 }
 
+// PersistentVolumeClaimInfoStatus contains the relavant information virt-handler needs cached about a PVC
+// +k8s:openapi-gen=true
+type PersistentVolumeClaimInfoStatus struct {
```

Comment:
`*InfoStatus` seems kind of redundant, maybe `PersistentVolumeClaimInfo` or `PersistentVolumeClaimStatus`?

Reply:
pr looks good to me otherwise

Reply:
yup i agree. I changed the struct name to PersistentVolumeClaimInfo